src/components/FSHControls.js

```diff
@@ -139,7 +139,12 @@ export default function FSHControls(props) {
     props.resetLogMessages();
     props.onSUSHIClick(true, [''], true);
     const dependencyArr = sliceDependency(dependencies);
-    const config = { canonical, version, FSHOnly: true, fhirVersion: ['4.0.1'] };
+    const config = {
+      canonical: canonical ? canonical : 'http://example.org',
```

Comment:
Since it looks like the default value used by GoFSH is `sample.org` (see [here](https://github.com/FHIR/GoFSH/blob/e664582d8587109431fefff430ff877cedc5b079/src/extractor/ConfigurationExtractor.ts#L10)), I think we should just use `sample.org` here. It is a bit confusing if you put something like an Instance on the RHS, and then turn it into FSH, and see that now the configuration is populated with `sample.org`, when the default is `example.org`.

Reply:
I actually prefer `example.org`.  For one, it resolves to a more appropriate web site than `sample.org` (since `example.org` is actually owned and managed by IANA specifically to support use cases like this).

So I'd suggest that if we want to align better (and I think we do, so good call, @ngfreiter), then we should update GoFSH to use `example.org` instead.  Perhaps @jafeltra can do that as part of her other updates to GoFSH.